python/README.md

```diff
@@ -34,14 +44,25 @@ gcloud dataproc clusters create ${CLUSTER_NAME} \
     --initialization-actions gs://goog-dataproc-initialization-actions-${REGION}/python/pip-install.sh
 ```
 
-## conda install packages
+## Install Conda packages
 
-Install conda packages into user python environment. The full list of packages is specified
-as `CONDA_PACKAGES` metadata key. Packages are space separated and can contain version selectors.
+Install Conda packages into user Python environment using `conda` command.
 
-Example 1: installing one package at head
+**Note:** when using this initialization action with automation, pinning package
+versions (see examples) is strongly encouraged to make cluster environment
+hermetic.
 
-```
+### Options
+
+-   `CONDA_CHANNELS` - a space separated list of channels to configure.
```

Comment:
Will packages be installed only from these channels? will the default channel be used?

Reply:
Added clarification.